js/components/PortfolioItem.jsx

```diff
@@ -20,14 +20,21 @@ var PortfolioItem = React.createClass({
       </div>;
     }
     var name = this.props.category.category;
+    var repsAvailable = '';
```

Comment:
Determines whether or not to show the private fields for the portfolio table. Very similar to the categories table